Extensions/find_inactives.js

```diff
@@ -283,6 +284,22 @@ XKit.extensions.find_inactives = new Object({
 
 		});
 
+		$(".unfollow_button").bind("click", function() {
+			console.log(this);
+			var unfollowButton = $(this);
+			var name = $(this).attr("data-name");
```

Comment:
use var unfollowButton instead of $(this) to avoid jquery variable initialization overhead